2019/PaperCall.md

```diff
@@ -49,25 +57,70 @@ We expect participants to follow these rules at conference and workshop venues a
+
+Your proposals should include as many details as possible about the topic and format for the presentation. Vague and overly broad proposals don’t showcase your skills and knowledge. The more you can tell us, the more likely the proposal will be selected.
 ```
 
 ## Additional Information
 
-_This field supports Markdown. This is where you can request additional information from your speakers. If you leave this field blank, it won't show up on the CFP._
+_This is where you can request additional information from your speakers. If you leave this field blank, it won't show up on the CFP._
 
 ```
-TODO
+* Please tell us your name
+* Please provide a link to a video of a previous speaking engagement
+* Please share your twitter, github link, your personal website or blog
+* Please write your short bio
+* Your employment & position
+* Your e-mail address
+* Do you require any special meals, assistance, etc?
+* What is your T-shirt size?
+* Can your company cover your travel & accommodation costs? (we grant partner status to companies who do that)
 ```
 
 ## CFP Notes
 
-_ This field supports Markdown. Here is where you should put any miscellaneous information about your conference-links, contact information, etc._
+_ Here is where you should put any miscellaneous information about your conference-links, contact information, etc._
+
+```
+If you have any questions, please, email team@bkug.by 
+```
+
+## Do you offer travel assistance?
+
+- [x] Yes
+
+## If yes, what is your policy?
+   
+_Please describe what your travel policy for speakers is. Do you offer flights, accommodation, nothing? Please let them know._
 
 ```
-TODO
+We are doing our best to reimburse flight and accommodation. However our ability is limited. So, if you feel that free conference ticket is enough to join us as a speaker, or do you have company that can cover your travel expenses, please let us know. We will for sure have to manage carefully expenses, so any help from your side will be greatly appreciated. At the same time we will do our best to reimburse as many speakers’ expenses as possible. Please let us know which statement below applies to you: 
+
+* I need flight and/or accommodation reimbursement 
+* It would be great to have flight and/or accommodation reimbursement, but I will come on my own if needed 
+* I will fly on my own or my company will cover flight. Free conference ticket is enough for me.
+
+If you fly to Minsk airport from any country except Russia & your stay will last up to 30 days (including arrival & departure dates), the visa will be stamped to you free of charge at Minsk airport!
+
+This concerns [74 countries' citizens](http://mfa.gov.by/en/visa/visafreetravel/e0ced19bb1f9bf2c.html).
 ```
 
 ## Talk Tags
 
 Kotlin, Google Cloud Platform, Android, Kotlin/Native, Kotlin/JS
+
+## Which types of talks will you accept?
+
+- [x] Talk (~30-45 minutes)
+- [x] Lightning Talk (~5-10 minutes)
+- [x] Workshop (> 60 minutes)
```

Comment:
The same question

Reply:
Code labs (workshops, etc) recommended format according Kotlin/Everywhere guide. Since many speakers do talks and workshops I think speakers can submit their workshops as well, and then we decide to do them, or not.

Reply:
ok